helm/templates/role.yaml

```
@@ -9,3 +9,10 @@ rules:
 - apiGroups: ["*"]
   resources: ["deployments", "pods", "replicacontrollers", "replicasets", "daemonsets", "statefulsets", "jobs", "cronjobs", "imagepolicies", "mutatingwebhookconfigurations"]
   verbs: ["get"]
+{{- if .Values.psp.enabled }} 
+- apiGroups: ['policy']
+  resources: ['podsecuritypolicies']
+  verbs:     ['use']
+  resourceNames:
+  - {{ .Values.psp.name }}
+{{- end}}
```

```suggestion
{{- if .Values.psp }}
{{- if .Values.psp.enabled }} 
- apiGroups: ['policy']
  resources: ['podsecuritypolicies']
  verbs:     ['use']
  resourceNames:
  - {{ .Values.psp.name }}
{{- end}}
{{- end}}
```
when implemented like this, not providing any `psp` entry in `helm/values.yaml` does not cause an error which is somewhat more fail safe.